AirCasting/SearchAndFollow/CompleteScreen/SearchAndFollowChartViewModel.swift

```diff
@@ -0,0 +1,21 @@
+// Created by Lunar on 28/02/2022.
+//
+
+import Foundation
+import Charts
+
+class SearchAndFollowChartViewModel: ObservableObject {
+    @Published var entries: [ChartDataEntry] = []
+    
+    init(stream: SearchSession.SearchSessionStream) {
+        generateEntries(for: stream)
+    }
+    
+    func generateEntries(for stream: SearchSession.SearchSessionStream) {
```

Comment:
same as before - `generateEntries` is not a user interaction so it should not be a public method on a viewmodel. It also takes a data object as a parameter which the view should not know about (except for passing it from its init to a viewmodel init in SwiftUI approach)